html/javascript/utils.js

```diff
@@ -84,8 +84,11 @@ _crgUtils = {
 			close: function() { if (callback('default'))
 				dialog.dialog("destroy");
 			}
-		});
-	},
+    });
+    // If we're in period 2, the dialog can end up off screen once all the
+    // SK rows finish loading. This ensures the dialog stays up top.
+    dialog.dialog("widget").css({"top": "5em", "position": "fixed"});
+  },
```

Comment:
It looks like indentation is inconsistent with the rest of the file.

Reply:
Ah, I did the replacement the wrong way around. Fixed.

Reply:
The rows with brackets are still indented by spaces

Reply:
That should do it...